helix-term/src/ui/prompt.rs

```diff
@@ -446,18 +449,24 @@ impl Prompt {
             for (i, (_range, completion)) in
                 self.completion.iter().enumerate().skip(offset).take(items)
             {
-                let color = if Some(i) == self.selection {
-                    selected_color // TODO: just invert bg
+                let is_selected = Some(i) == self.selection;
+
+                let completion_item_style = if is_selected {
+                    selected_color
+                } else if completion.style != Style::default() {
+                    completion.style
                 } else {
-                    completion_color
+                    default_completion_color
```

Comment:
Here you can use `completion_color.patch(completion.style)` for roughly the same effect

Reply:
neat! Added in https://github.com/helix-editor/helix/pull/12205/commits/cefa10da700c8712d2f9d444c8eea89f32b688f4